fix for "stropts.h: No such file or directory"
see also https://github.com/tensorflow/tensorflow/pull/14055

From 0cd4a9b048c40ce2597746e55b44f9dfc2e7b647 Mon Sep 17 00:00:00 2001
From: Toon Verstraelen <Toon.Verstraelen@UGent.be>
Date: Sat, 28 Oct 2017 05:07:55 +0200
Subject: [PATCH] Do not use stropts in curl

---
 third_party/curl.BUILD | 1 -
 1 file changed, 1 deletion(-)

diff --git a/third_party/curl.BUILD b/third_party/curl.BUILD
index 882967df1c1..805a30d2620 100644
--- a/third_party/curl.BUILD
+++ b/third_party/curl.BUILD
@@ -477,7 +477,6 @@ genrule(
         "#  define HAVE_RAND_EGD 1",
         "#  define HAVE_RAND_STATUS 1",
         "#  define HAVE_SSL_GET_SHUTDOWN 1",
-        "#  define HAVE_STROPTS_H 1",
         "#  define HAVE_TERMIOS_H 1",
         "#  define OS \"x86_64-pc-linux-gnu\"",
         "#  define RANDOM_FILE \"/dev/urandom\"",
